Drop-In/src/main/res/layout/bt_fragment_select_payment_method.xml

```diff
@@ -0,0 +1,123 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_margin="0dp"
+    android:onClick="onBackgroundClicked"
+    android:background="@android:color/transparent"
+    android:layoutAnimation="@anim/bt_bottom_sheet_layout_animation"
+    tools:context="com.braintreepayments.api.DropInActivity">
+
+    <LinearLayout
+        android:id="@+id/bt_dropin_bottom_sheet"
+        android:elevation="5dp"
+        android:layout_width="@dimen/bt_full_width"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:layout_alignParentBottom="true"
+        android:layout_centerHorizontal="true"
+        android:animateLayoutChanges="true"
+        android:background="@color/bt_base_background"
+        tools:targetApi="lollipop">
```

Comment:
I think this can be removed since Lollipop is our minSdkVersion now.

Reply:
Good call! Will remove.